src/librustc_lexer/src/lib.rs

```diff
@@ -236,7 +236,7 @@ pub enum Base {
 /// (e.g. "#![deny(missing_docs)]").
 pub fn strip_shebang(input: &str) -> Option<usize> {
     debug_assert!(!input.is_empty());
-    if !input.starts_with("#!") || input.starts_with("#![") {
+    if !input.starts_with("#!") || input.starts_with("#![") || input.starts_with("#! [") {
```

Comment:
This doesn't cover a case that has two or more white spaces.

Reply:
Done